assets/javascript/app.js

```diff
@@ -0,0 +1,214 @@
+    choices: [
+      "Robert Mondavi",
+      "Rodney Strong",
+      "Charles Krug",
+      "Georges de Latour",
+      "André Tchelistcheff"
+    ],
+    answer: "Georges de Latour"
+  },
+  {
+    question:
+      "Which American appelation is best known for producing a dry and 'dusty' red grape?",
+    choices: [
+      "Calistoga",
+      "Los Carneros",
+      "Rutherford",
+      "Stags Leap",
+      "Sonoma Valley"
+    ],
+    answer: "Rutherford"
+  },
+  {
+    question: "Which grape is looked down upon as being 'inferior'?",
+    choices: ["Merlot", "Sauvignon Blanc", "Chardonnay", "Cabernet Sauvignon"],
+    answer: "Chardonnay"
+  },
+  {
+    question: "Which description is generally NOT used to describe wine?",
+    choices: [
+      "dry",
+      "fruit-forward",
+      "tanic",
+      "bold",
+      "peppery",
+      "savory",
+      "earthy",
+      "barnyardy"
+    ],
+    answer: "savory"
+  },
+  {
+    question:
+      "Which country is the largest producer of the Sauvignon Blanc grape?",
+    choices: ["France", "Italy", "Australia", "United States", "South Africa"],
+    answer: "France"
+  },
+  {
+    question:
+      "In which month are grapes harvested and 'crushed' in Napa Valley?",
+    choices: ["August", "September", "October", "November", "December"],
+    answer: "October"
+  }
+];
+
+// setup for interval timer beginning with some global variables - 
+
+var box = $("#question-box");
+var countdown;
+
+// could have made this whole process simpler without the use of writing functions with object notation
+var gameOutcome = {
+  correct: 0,
+  incorrect: 0,
+  interval: 180,
+
+  //   this function decrements interval starting from 180 seconds -
+
+  clock: function() {
+    gameOutcome.interval--;
+    $("#counter").html(gameOutcome.interval);
+    if (gameOutcome.interval === 0) {
+      console.log("Game Over");
+      gameOutcome.done();
+    }
+  },
+
+  //   still working on how/where to call function as to reset the game (see lines 140, which empty's box at end and lines 209-211)
+
+  start: function() {
+    countdown = setInterval(gameOutcome.clock, 1000);
+    // "this" is referring to the gameOutcome object (correct & incorrect are properties of the object)
+    this.correct = 0;
+    this.incorrect = 0;
+    $("#question-box").empty();
+    $("#question-box").prepend(
+      "<p>Time Remaining: <span id='counter'>180</span> Seconds</p>"
+    );
+    $("#start").remove();
+
+    // this loops through all the questions in questions array object
+
+    for (var i = 0; i < questions.length; i++) {
+      box.append("<h4>" + questions[i].question + "</h4>");
+      for (var j = 0; j < questions[i].choices.length; j++) {
+        box.append(
+          "<input type='radio' name='question-" +
+            i +
+            "' value='" +
+            questions[i].choices[j] +
+            "''>" +
+            questions[i].choices[j]
+        );
```

Comment:
You may want to add a `<br/>` tag so that the choices are on a new line. Or, use some CSS on the resulting inputs. It makes the choices more legible on the page.